<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/img/bg.png</normaloff>:/img/img/bg.png</iconset>
  </property>
  <widget class="QLabel" name="tempLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/img/login.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>571</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LoginLayoutBox">
    <item>
     <layout class="QHBoxLayout" name="userNameLayoutBox">
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Username: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameInput">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="passwordLayoutBox">
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Password: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordInput">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="onOkButton">
      <property name="text">
       <string>Login</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/img/login.png</normaloff>:/img/img/login.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="userNameError">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="incorrectAnything">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="passwordError">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>290</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-5</y>
     <width>1011</width>
     <height>491</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/img/background.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="loginscreen">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;LOGIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>tempLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>userNameError</zorder>
  <zorder>incorrectAnything</zorder>
  <zorder>passwordError</zorder>
  <zorder>loginscreen</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
